web_src/js/components/DashboardRepoList.vue

```diff
@@ -425,6 +423,10 @@ const sfc = {
       return 'octicon-repo';
     },
 
+    repoIconSize(repo) {
+      return repo.mirror ? 14 : 16; // octicon-mirror does not have padding
+    },
+
```

Comment:
I think pure CSS could also do the job, something like this (if it really needs to do so ....)

```
.repo-list-icon.octicon-mirror {
}
```

Reply:
Done.